applications/dashboard/models/class.usermodel.php

```diff
@@ -1327,21 +1327,29 @@ public function getApplicantCount() {
     /**
      * Returns all users in the applicant role.
      *
+     * @param int|bool $limit
```

Comment:
Just int|false here would be better.
You can actually pass `true` but that would be dumb.

Reply:
My IDE complains when I give it a specific value, instead of a type. :frowning_face:

Reply:
Welp. Mine doesn't. No big deal..